src/containers/Dashboard/Quizzes/Quiz.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component, Fragment } from "react";
+import "./Quiz.scss";
+import QuizCard from "./QuizCard";
+import { addNotification } from "../../../utilities";
+import API from "../../../api/index";
+
+class Quizzes extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      quizzes: [],
+      quiz : []
+    };
+  }
+
+  componentWillMount() {
+    API.getQuizzes(result => {
+      if (result.status === "200") {
+        this.setState({
+          quizzes: result.data
+        });
+        if (this.state.quizzes.length < 6) {
+          this.state.quizzes.map((quiz, index) => {
+            this.setState({
+              quiz: this.state.quiz.concat(quiz.name)
+            });
+          })
+        } else {
+          for(let i =0 ; i<6; i++) {
+            this.setState({
+              quiz: this.state.quiz.concat(this.state.quizzes[i].name)
+            });
+          }
+        }
+      } else {
+        // addNotification(
+        //   this.notificationDOMRef,
+        //   "Error",
+        //   "danger",
+        //   result.message
+        // );
+      }
+    }).catch = error => {
+      addNotification(this.notificationDOMRef, "Error", "warning", error);
+    };
+  }
+
+  render() {
+    return (
+      <Fragment>
+        <div>
+          <h5 className="headings ml-5 mb-5 mt-5">Quizzes</h5>
+        </div>
+        <div className="row">
+          <div className="quiz-container">
+          {this.state.quiz.map((quiz, index) => (
+            <div className="interview-prep-quiz" key={index}>
+            <div className="quiz-card shadow-lg ml-5 mr-4 mb-5">
+              <QuizCard heading={quiz}/>
```

Comment:
why are we passing just name to the quiz where is the description why not object is passing here

Reply:
Done